src/components/home/VenturesPhotos/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import Img from "gatsby-image"
+import { StaticQuery, graphql } from "gatsby"
+
+import "./index.module.css"
+
+const TeamPhotos = props => {
```

Comment:
Why not destructure `props` in the declaration?

Reply:
Also, I don't get the name `TeamPhotos`, can you explain the reasoning?

Reply:
Copy pasta